imei-svtav1.sh

```
@@ -0,0 +1,95 @@
+              if [ -n "$INSTALL" ]; then
+                make uninstall
+              fi
+        else
+          make install
+        fi
+
+        ldconfig $LIB_DIR
+      fi
+    } >>"$LOG_FILE" 2>&1
+  }; then
+    UPDATE_LIBHEIF="yes"
```

Shellcheck

[warning] UPDATE_LIBHEIF appears unused. Verify use (or export if used externally).

(SC2034)